pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -58,12 +58,29 @@
 # specific to instance because deepinheritance differs between class and
 # instance operations.
 deepinheritance_option = [              # pylint: disable=invalid-name
-    click.option('-d', '--deepinheritance', is_flag=True, required=False,
+    click.option('-d', '--deep-inheritance', is_flag=True, required=False,
                  help='If set, requests server to return properties in '
                       'subclasses of the target instances class. If option not '
                       'specified only properties from target class are '
                       'returned')]
 
+localonlyget_option = [              # pylint: disable=invalid-name
+    click.option('-l', '--local-only', is_flag=True, required=False,
+                 help='Request that server show only local properties of the '
+                      'returned instance. Some servers may not process this '
+                      'parameter.')]
+
+
+localonlyenum_option = [              # pylint: disable=invalid-name
+    click.option('-l', '--local-only', is_flag=True, required=False,
+                 help='Show only local properties of the instances. This '
+                      'subcommand may use either pull or traditional '
+                      'operations depending on the server and the '
+                      '"--use--pull-ops" general option. If pull operations '
```

Comment:
Show the "--use--pull-ops" option without quotes around it, and remove the extra dash inside of the name.

Reply:
Agreed. DONE